.github/workflows/test.yml

```diff
@@ -38,7 +38,12 @@ jobs:
           forge build
         id: build
 
-      - name: Run Forge tests
+      - name: Run offline Forge tests
         run: |
-          forge test -vvv
+          forge test --no-match-contract '.*ForkedTest$' -vvv
         id: test
+
+      - name: Run foked Forge tests
+        run: |
+          forge test --fork-url "https://eth.llamarpc.com" --match-contract '.*ForkedTest$' -vvv
```

Comment:
I'd suggest here that we pick a fixed block from which to fork from, i.e. `--fork-block-number <BLOCK>`, so that the tests are reproducible (this would also mean that we should convey in a readme that the RPC used should be of an archive-type. From memory, [`eth.merkle.io`](https://merkle.io/free-eth-rpc) provides archive data). See: https://book.getfoundry.sh/reference/forge/forge-test#forking

Reply:
Done, wdyt?